Lib/idlelib/idle_test/test_tooltip.py

```diff
@@ -65,81 +69,92 @@ class HovertipTest(unittest.TestCase):
     def setUp(self):
         self.top, self.button = _make_top_and_button(self)
 
+    def is_tipwindow_shown(self, tooltip):
+        return tooltip.tipwindow and tooltip.tipwindow.winfo_viewable()
```

Comment:
I might have used an even shorten name ('tip_shown'), but even as is, this is a definite improvement in readability.